openapi.yaml

```diff
@@ -139,54 +139,72 @@ info:
-    | Access-Control-Allow-Methods     | Comma-separated list of HTTP methods allowed to be requested. Back-ends MUST list all implemented HTTP methods for the endpoint here. | `OPTIONS, GET, POST, PATCH, PUT, DELETE` |
+    | Access-Control-Allow-Origin      | Allowed origin for the request, including protocol, host and port or `*` for all origins. It is RECOMMENDED to return the value `*` to allow requests from browser-based implementations such as the Web Editor. | `*` |
     | Access-Control-Expose-Headers    | Some endpoints send non-safelisted HTTP response headers such as `OpenEO-Identifier` and `OpenEO-Costs`. All headers except `Cache-Control`, `Content-Language`, `Content-Type`, `Expires`, `Last-Modified` and `Pragma` must be listed in this header. Currently, the openEO API requires at least the following headers to be listed: `Location, OpenEO-Identifier, OpenEO-Costs, Link`. | `Location, OpenEO-Identifier, OpenEO-Costs, Link` |
-    | Content-Type                     | SHOULD return the content type delivered by the request that the permission is requested for. | `application/json` |
+    
+    
+    **Tip**: Most server can send the required headers and the responses to the OPTIONS requests globally. Otherwise you may want to use a proxy server to add the headers and OPTIONS responses.
     
     ### Example request and response
     
     Request:
     
     ```http
-    OPTIONS /api/v1/jobs HTTP/1.1
+    POST /api/v1/jobs HTTP/1.1
     Host: openeo.cloudprovider.com
-    Origin: http://client.org:8080
-    Access-Control-Request-Method: POST 
-    Access-Control-Request-Headers: Authorization, Content-Type
+    Origin: https://client.org:8080
+    Authorization: Bearer basic//ZXhhbXBsZTpleGFtcGxl
     ```
     
     Response:
     
     ```http
-    HTTP/1.1 204 No Content
-    Access-Control-Allow-Origin: http://client.org:8080
-    Access-Control-Allow-Credentials: true
-    Access-Control-Allow-Methods: OPTIONS, GET, POST, PATCH, PUT, DELETE
-    Access-Control-Allow-Headers: Authorization, Content-Type
+    HTTP/1.1 201 Created
+    Access-Control-Allow-Origin: *
+    Access-Control-Expose-Headers: Location, OpenEO-Identifier, OpenEO-Costs, Link
     Content-Type: application/json
+    Location: https://openeo.cloudprovider.com/api/v1/jobs/abc123
+    OpenEO-Identifier: abc123
     ```
     
-    ## CORS headers
+    ## OPTIONS method
     
-    The following headers MUST be included with every response:
+    All endpoints must respond to the `OPTIONS` HTTP method. This is a response for the preflight requests made by web browsers before sending the actual request (e.g. `POST /jobs`). It needs to respond with a status code of `204` and no response body. In addition to the HTTP headers shown in the table above, the following HTTP headers MUST be included with every response to an `OPTIONS` request:
```

Comment:
> In addition to the HTTP headers shown in the table above

"In addition to the general CORS headers shown in the table above."

To make it less likely that readers fail to see that this an addition: put "In addition" in italic, or start a new paragraph for this sentence? (case in point: I just realized we there are two CORS header tables in our spec :smile:  )

Reply:
Done